### PR TITLE
Contribute: adds template for issues

### PR DESCRIPTION
**Pull request by vanemoraess**
ref: https://github.com/ada-url/ada/issues/199

Hey everyone! Please let me know if this is what is expected for an "issue template".

### CODE REVIEW

File: contribute/issue-template.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Please describe the expected behavior.
+
+### Current Behavior
+
+Please describe the current behavior.
+
+### Screenshots
+
+If possible, attach screenshots that may help clarify the problem.
```
Comment (anonrig) [inline contribute/issue-template.md:33]: I don't think screenshots should be included in the bug report.

**Review by anonrig — COMMENTED** [review]
Thanks for the contribution. Can you also add a part where users can provide their operating system etc. @lemire What kind of information would be beneficial for pinpointing a bug?

File: contribute/issue-template.md (hunk 1 above)
Comment (vanemoraess) [reply contribute/issue-template.md]: Removed!

**Review by anonrig — APPROVED** [review]

**Comment by anonrig** [comment]
Thank you for the contribution!

**Pull request merged**